server/handler/query.go

```diff
@@ -128,9 +129,10 @@ func Query(db service.SQLDB) RequestProcessFunc {
 }
 
 func killQuery(db service.SQLDB, query string) {
-	pRows, pErr := db.Query("SHOW FULL PROCESSLIST")
+	const showProcessList = "SHOW FULL PROCESSLIST"
+	pRows, pErr := db.Query(showProcessList)
 	if pErr != nil {
-		// TODO (carlosms) log error when we migrate to go-log
+		log.Errorf(pErr, "failed to execute %q", showProcessList)
```

Comment:
it's better to use logger from context. Because currently, it's unclear from logs which request triggered error (in hosted version).

Reply:
Done in d07e77a